src/components/logIn/LogIn.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+  <div class="form wrapper">
+    <v-sheet color="white" elevation="5" rounded>
+      <v-form ref="form" v-model="valid" @submit.prevent="submit">
+        <v-text-field
+          v-model="email"
+          :counter="50"
+          :rules="emailRules"
+          :error-messages="errors.email"
+          prepend-icon="mdi-email"
+          label="E-mail"
+          background-color="deep-purple lighten-4"
+          required
+          solo
+          clearable
+        ></v-text-field>
+
+        <v-text-field
+          v-model="password"
+          :counter="50"
+          :rules="passwordRules"
+          :error-messages="errors.password"
+          prepend-icon="mdi-lock"
+          type="password"
+          label="Password"
+          color="deep-purple lighten-4"
+          background-color="deep-purple lighten-4"
+          required
+          solo
+          clearable
+        ></v-text-field>
+        <div class="button wrapper">
+          <v-btn
+            @click="validate"
+            :disabled="!valid"
+            class="mr-4"
+            type="submit"
+            color="success"
+          >
+            submit
+          </v-btn>
+          <v-btn @click="redirectToWelcomePage" color="deep-purple lighten-5">
+            back
+          </v-btn>
+        </div>
+      </v-form>
+    </v-sheet>
+    <v-snackbar
+      v-model="snackbar"
+      vertical="vertical"
+      transition="fab-transition"
+      :timeout="snackbarTimeout"
```

Comment:
use directives that start with _:_ before other directives

Reply:
Done